src/markerclusterer.js

```diff
@@ -1672,3 +1672,5 @@ if (typeof String.prototype.trim !== 'function') {
   };
 }
 
+module.exports = MarkerClusterer;
```

Comment:
It does throw an exception error but will work fine either way. I opted not to go for full UMD style simply because I figured it'd be outside the scope of this pull request. I wanted to add `module.exports` only.

I'll add an `if` though, checking for module so it won't throw that exception!